modules/stanford_person_views/stanford_person_views.views_default.inc

```diff
@@ -324,6 +324,141 @@ function stanford_person_views_views_default_views() {
   $handler->display->display_options['fields']['field_s_person_affiliation']['type'] = 'taxonomy_term_reference_plain';
   $handler->display->display_options['fields']['field_s_person_affiliation']['group_rows'] = FALSE;
   $handler->display->display_options['fields']['field_s_person_affiliation']['delta_offset'] = '0';
+  $handler->display->display_options['defaults']['sorts'] = FALSE;
+  /* Sort criterion: Taxonomy term: Weight */
+  $handler->display->display_options['sorts']['weight']['id'] = 'weight';
+  $handler->display->display_options['sorts']['weight']['table'] = 'taxonomy_term_data';
+  $handler->display->display_options['sorts']['weight']['field'] = 'weight';
+  $handler->display->display_options['sorts']['weight']['relationship'] = 'term_node_tid_1';
+  /* Sort criterion: Content: Sorting Weight (field_s_person_weight) */
+  $handler->display->display_options['sorts']['field_s_person_weight_value']['id'] = 'field_s_person_weight_value';
+  $handler->display->display_options['sorts']['field_s_person_weight_value']['table'] = 'field_data_field_s_person_weight';
+  $handler->display->display_options['sorts']['field_s_person_weight_value']['field'] = 'field_s_person_weight_value';
+  /* Sort criterion: Content: Last Name (field_s_person_last_name) */
+  $handler->display->display_options['sorts']['field_s_person_last_name_value']['id'] = 'field_s_person_last_name_value';
+  $handler->display->display_options['sorts']['field_s_person_last_name_value']['table'] = 'field_data_field_s_person_last_name';
+  $handler->display->display_options['sorts']['field_s_person_last_name_value']['field'] = 'field_s_person_last_name_value';
+  /* Sort criterion: Content: First name (field_s_person_first_name) */
+  $handler->display->display_options['sorts']['field_s_person_first_name_value']['id'] = 'field_s_person_first_name_value';
+  $handler->display->display_options['sorts']['field_s_person_first_name_value']['table'] = 'field_data_field_s_person_first_name';
+  $handler->display->display_options['sorts']['field_s_person_first_name_value']['field'] = 'field_s_person_first_name_value';
+  $handler->display->display_options['path'] = 'people/%/cap-list/grouped/affiliation';
+
+  /* Display: Group by Student Page */
+  $handler = $view->new_display('page', 'Group by Student Page', 'page_2');
```

Comment:
The title for this view is Student but this view exists at `people/%/cap-list/grouped`